src/Symfony/Component/Notifier/Bridge/Pusher/README.md

```diff
@@ -0,0 +1,32 @@
+Pusher Notifier
+==============
+
+Provides [Pusher](https://pusher.com) integration for Symfony Notifier.
+
+DSN example
+-----------
+
+```
+PUSHER_DSN=pusher://APP_KEY:APP_SECRET@APP_ID?server=SERVER
+```
+
+where:
+
+- `APP_KEY` is your app unique key
+- `APP_SECRET` is your app unique and secret password
+- `APP_ID` is your app unique id
+- `SERVER` is your app server
```

Comment:
```suggestion
where:

 - `APP_KEY` is your app unique key
 - `APP_SECRET` is your app unique and secret password
 - `APP_ID` is your app unique id
 - `SERVER` is your app server
```

Reply:
this does not make sense to me. Markdown lists don't require indenting the bullet markers (and if you indent them too much, you end up creating a code block once you reach the minimum indentation of a code block)

Reply:
IIRC there was a PR in the past by @nicolas-grekas 

I agree with the codeblock, but one space should be no problem